projects.json

```diff
@@ -4763,7 +4763,19 @@
         "action": "BuildSwiftPackage",
         "configuration": "release",
         "build_tests": "true",
-        "tags": "swiftpm"
+        "tags": "swiftpm",
+        "xfail": [
+          {
+            "issue": "https://github.com/apple/swift-package-manager/issues/7521",
+            "compatibility": ["5.9"],
+            "branch": ["release/5.9"],
```

Comment:
Is branch here not referring to the toolchain branch? ie. release/5.9 is fine isn't it? The problem is main (and release/6.0 when we cherry-pick the changes there).

Reply:
Hm, right, I was confused by the names of those jobs, looks like a job called 5.9 is actually testing with `main` snapshots or fresh builds? That's why I just followed what other xfails are doing.